src/compute-client/src/controller/instance.rs

```diff
@@ -751,8 +753,9 @@ where
 
     /// Update the max size in bytes of any result.
     pub fn update_max_result_size(&mut self, max_result_size: u32) {
```

Comment:
I am wondering whether this PR would be the right one to update the public interface offered here to a more general `update_configuration`? Since internally we would be ready to deal with extensibility in configuration settings, perhaps this should percolate up to the callers?

Reply:
This seems like a good idea! I think I would rather do that in a separate PR though, as we should also adjust the `create_instance` API method accordingly (from taking a `max_result_size` argument to taking a `config` argument instead).

Reply:
Agreed!